packages/forge/blocks/posthog/src/logo.tsx

```diff
@@ -0,0 +1,53 @@
+/** @jsxImportSource react */
+
+export const PosthogLogo = (props: React.SVGProps<SVGSVGElement>) => (
+  <svg width="25px" height="15px" viewBox="0 0 25 15" version="1.1">
```

Comment:
```suggestion
  <svg viewBox="0 0 25 15" {...props}>
```

This only need viewbox and spread the props to be displayed properly

Reply:
Done